vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -796,21 +796,44 @@ public boolean isSvod() {
     }
 
     /**
-     * Helper to determine if the video is part of the SVOD library and that the user currently has access to it.
-     * This will be true if the user has purchased an SVOD subscription that is active.
+     * Helper to determine if this video is theoretically playable. Note that there may be device limitations
+     * that cause a video to fail to play back in practice.
      *
-     * @return true if this video can be accessed with an SVOD subscription and the user's subscription is active
+     * @return true if this video can be played, false otherwise
      */
-    public boolean canAccessVideoInSvod() {
-        return getSvodPurchaseDate() != null;
+    public boolean isPlayable() {
+        return getPlayStatus() == Play.Status.PLAYABLE;
     }
 
+    /**
+     * Helper to determine if this is a trailer that is associated with an SVOD title
+     *
+     * @return true if this is a trailer that is associated with an SVOD title, false otherwise
+     */
+    public boolean isSvodRelatedTrailer() {
+        return isTrailer() && getSvodInteraction() != null;
```

Comment:
Since the method is named `isSvodRelatedTrailer `, a suggestion here would be to write this as `isTrailer() && isSvod()`. This way if the logic behind `isSvod()` changes from `getSvodInteractins() != null`, we are a little more protected.